etc/behat/services.xml

```diff
@@ -23,6 +23,9 @@
         <parameter key="sylius.behat.table_manipulator.class">Sylius\Behat\TableManipulator</parameter>
         <parameter key="sylius.behat.page.class">Sylius\Behat\Page\Page</parameter>
         <parameter key="sylius.behat.symfony_page.class">Sylius\Behat\Page\SymfonyPage</parameter>
+        <parameter key="sylius.behat.page.admin.crud.index.class">Sylius\Behat\Page\Admin\Crud\IndexPage</parameter>
+        <parameter key="sylius.behat.page.admin.crud.create.class">Sylius\Behat\Page\Admin\Crud\CreatePage</parameter>
+        <parameter key="sylius.behat.page.admin.crud.edit.class">Sylius\Behat\Page\Admin\Crud\EditPage</parameter>
```

Comment:
For now we use **update** as a name of action and routes suffix for both _displaying form_ and _updating resource_. As @pjedrzejewski mention in whole codebase, we use update to express everything around update. For me is more important to be consistent with the codebase rather than on feeling what is more appropriate. Action is with _an update_ keyword, route is with _an update_ word, so page should also be with _update_ keyword.